assets/mask-shapes/arrow.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 127.75 81.62"><defs><style>.cls-1{fill:#1b1b1a;}</style></defs><polygon class="cls-1" points="70.68 26.54 70.68 0 35.34 20.4 0 40.81 35.34 61.21 70.68 81.61 70.68 55.08 127.75 55.08 127.75 26.54 70.68 26.54"/></svg>
```

Comment:
Do we need the `id="Layer_1" data-name="Layer 1"`? the `cls-1` CSS & class name?

Reply:
Yeah, I should remove it, but the shapes will get changed anyway.